client/Bootstrap/index.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import '@/Styling';
+import { Home } from '@/Pages';
+import './i18n';
+import ReactDOM from 'react-dom';
+import React from 'react';
+
+const onLoad: () => void = () => {
```

Comment:
Still on browser load? Was expecting index would be a script which fires as soon as it can to (start) rendering the UI

Reply:
As we are starting to add actual react code (even placeholder code at that), I think it is time to switch off browser load - it should fire as soon as it is loaded.